### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: changelog.d/+bump-dependencies.changed.md
Comment (Razz4780) [inline changelog.d/+bump-dependencies.changed.md:1]: `.internal` maybe?

**Review by Razz4780 — COMMENTED** [review]
What about `./mirrord/layer/tests/apps/issue834/go.mod`

File: changelog.d/+bump-dependencies.changed.md
Comment (aviramha) [reply changelog.d/+bump-dependencies.changed.md:1]: I did bump mirrord dependencies itself so it is a change :)

**Comment by aviramha** [comment]
> issue834/go.mod

not relevant to the scope of this upgrade (ssdlc security finding)

**Review by Razz4780 — APPROVED** [review]

**Pull request merged**